battles/tests/test_views.py

```diff
@@ -74,10 +74,12 @@ def setUp(self):
                                                battle_related=self.battle,
                                                pokemons=self.pokemons,
                                                trainer=self.battle.opponent)
-        self.view_url = reverse_lazy('battles:details',
-                                     kwargs={'pk': self.battle.id})
+        self.view_url = reverse_lazy('battles:details', kwargs={'pk': self.battle.id})
 
     def test_response_status_200(self):
+        self.battle = mommy.make('battles.Battle', creator=self.user)
```

Comment:
If you need this value to be really part of the class, it's better to add it to `setUp`, otherwise you can remove `self.`  reference here.